OfferSpace.BL/Models/Company.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace OfferSpace.BL.Models
+{
+    public class Company : IEntity<long>
+    {
+        public long Id { get; set; }
+        public string Name { get; set; }
+        public long CustomerId { get; set; }
```

Comment:
One company can have many customers and the filed and DB relationship is totally wrong, remove it.